app/controllers/foods_controller.rb

```diff
@@ -34,6 +34,6 @@ def destroy
   private
 
   def food_params
-    params.require(:food).permit(:name, :measurement_unit, :price, :user)
+    params.require(:food).permit(:name, :measurement_unit, :price, :user, :quantity)
```

Comment:
In the wireframe, the food index view page doesn't have quantity as part of it. It only appears in the recipe show page where you add ingredient
![Uploading image.png…]()
Please look at the first image